server/athenian/api/controllers/miners/github/precomputed_prs/__init__.py

```diff
@@ -0,0 +1,39 @@
+from athenian.api.controllers.miners.github.precomputed_prs.done_prs import \
+    delete_force_push_dropped_prs, DonePRFactsLoader, load_precomputed_done_candidates, \
+    load_precomputed_done_facts_all, load_precomputed_done_facts_filters, \
+    load_precomputed_done_facts_ids, load_precomputed_done_facts_reponums, \
+    load_precomputed_done_timestamp_filters, load_precomputed_pr_releases, \
+    store_precomputed_done_facts
+from athenian.api.controllers.miners.github.precomputed_prs.merged_prs import \
+    discover_inactive_merged_unreleased_prs, load_merged_pull_request_facts_all, \
+    load_merged_unreleased_pull_request_facts, MergedPRFactsLoader, \
+    store_merged_unreleased_pull_request_facts, update_unreleased_prs
+from athenian.api.controllers.miners.github.precomputed_prs.open_prs import \
+    OpenPRFactsLoader, store_open_pull_request_facts
+from athenian.api.controllers.miners.github.precomputed_prs.utils import \
+    build_days_range, remove_ambiguous_prs, triage_by_release_match
+
+
+__all__ = [
+    "DonePRFactsLoader",
+    "MergedPRFactsLoader",
+    "OpenPRFactsLoader",
+    "build_days_range",
+    "delete_force_push_dropped_prs",
+    "discover_inactive_merged_unreleased_prs",
+    "load_merged_pull_request_facts_all",
+    "load_merged_unreleased_pull_request_facts",
+    "load_precomputed_done_candidates",
+    "load_precomputed_done_facts_all",
+    "load_precomputed_done_facts_filters",
+    "load_precomputed_done_facts_ids",
+    "load_precomputed_done_facts_reponums",
+    "load_precomputed_done_timestamp_filters",
+    "load_precomputed_pr_releases",
+    "remove_ambiguous_prs",
+    "store_merged_unreleased_pull_request_facts",
+    "store_open_pull_request_facts",
+    "store_precomputed_done_facts",
+    "triage_by_release_match",
+    "update_unreleased_prs",
+]
```

Comment:
We don't `import *` here so this is useless.
```suggestion
```

Reply:
I did it to avoid the import unused errors rather than excluding the whole file from the `flake8` check.

Reply:
I see. I'll add a global exclusion in the config.